zenhappens/src/js/rangeslider.js

```diff
@@ -0,0 +1,42 @@
+// javascript by Trevan Hetzel on codepen
+// https://codepen.io/trevanhetzel/pen/rOVrGK
+
+var sheet = document.createElement('style'),  
+  $rangeInput = $('.range input'),
+  prefs = ['webkit-slider-runnable-track', 'moz-range-track', 'ms-track'];
```

Comment:
Add some variables for the lengths of time available:

```suggestion
  prefs = ['webkit-slider-runnable-track', 'moz-range-track', 'ms-track'],
  times = ['15 minutes', '30 minutes', '45 minutes', '60 minutes', '75 minutes', '90 minutes', '120 minutes'];
```